File: sct_source/SourceCode/API_SoCongThuong/Models/ElectricOperatingUnitsModel.cs
namespace API_SoCongThuong.Models
{
    public class ElectricOperatingUnitsModel
    {
        public Guid ElectricOperatingUnitsId { get; set; }
        public Guid CustomerName { get; set; }
        public string? CustomerString { get; set; }
        public string Address { get; set; } = null!;
        public string PhoneNumber { get; set; }
        public string PresidentName { get; set; }
        public string NumOfGP { get; set; } = null!;
        public string SignDay { get; set; } = null!;
        public Guid Supplier { get; set; }
        public string? SupplierName { get; set; }
        public bool IsPowerGeneration { get; set; }
        public bool IsPowerDistribution { get; set; }
        public bool IsConsulting { get; set; }
        public bool IsSurveillance { get; set; }
        public bool IsElectricityRetail { get; set; }
        public string? FieldActivity { get; set; }
        public int Status { get; set; }
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/electric-operating-units-page/_models/electric-operating-units.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface ElectricOperatingUnitsModel extends BaseModel {
  electricOperatingUnitsId: string;
  customerName: string;
  address: string;
  phoneNumber: string;
  presidentName: string;
  numOfGP: string;
  signDay: any;
  supplier: string;
  isPowerGeneration: boolean;
  isPowerDistribution: boolean;
  isElectricityRetail: boolean;
  isConsulting: boolean;
  isSurveillance : boolean;
  status: number;
}
